science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,237 @@
+---
+page-title: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-description: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-name: science-blog-6
+page-name_de: science-blog-6
+author: CWA-Team
+date: "2022-11-23"
+layout: science
+---
+
+In diesem Beitrag wollen wir untersuchen, welche Aussagekraft eine rote Warnung gegenüber einem niedrigen Risiko mit bzw. ohne Risikobegegnung hat. Hierzu betrachten wir den Positivenanteil der Testergebnisse, d.h. den Anteil der positiven Testergebnisse, in Abhängigkeit vom angezeigten Risiko. Die Daten hierfür stammen aus der [Datenspende der Corona-Warn-App](../2021-10-15-science-blog-4/#3-privacy-preserving-analytics).
```

Comment:
d.h => `d.&nbsp;h.`

Reply:
Hi @MikeMcC399  can you help me - what is your intention with the nbsp in between?

Thanks,
Sascha

Reply:
@mtb77 

> what is your intention with the nbsp in between?

It is about conforming to  [DIN 5008](https://de.wikipedia.org/wiki/DIN_5008#Abk%C3%BCrzungen_(Abschnitt_4.5)). The rest of the cwa-website is conforming to the correct way of writing German abbreviations.

"DIN 5008 regelt unter anderem Folgendes (die genannten Regelungen und Abschnittsnummern beziehen sich auf die Ausgabe 2011):

Abkürzungen (Abschnitt 4.5)
In [Abkürzungen](https://de.wikipedia.org/wiki/Abk%C3%BCrzung) aus mehreren Buchstaben, die jeweils einen Punkt haben und für ein Wort stehen, ist zwischen den abgekürzten Wörtern jeweils ein Leerzeichen zu setzen: a. a. O., d. h., v. l. n. r., z. B. usw. Um einen unerwünschten Zeilenumbruch innerhalb der Abkürzung zu vermeiden, sollte ein [geschütztes Leerzeichen](https://de.wikipedia.org/wiki/Gesch%C3%BCtztes_Leerzeichen) verwendet werden."

Reply:
Ok thanks - changed it!